sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -457,6 +466,14 @@ export type ElisionTokenFilter = BaseTokenFilter & {
 // @public
 export type EntityCategory = string;
 
+// @public
+export type EntityLinkingSkill = BaseSearchIndexerSkill & {
+    odatatype: "#Microsoft.Skills.Text.V3.EntityLinkingSkill";
+    defaultLanguageCode?: string | null;
```

Comment:
I always ask this, but does `null` have a distinct meaning from omitting the property, like to remove it on a PATCH operation?

Reply:
This update should have been done for this release. But, due to the time constraints spent in autorest, this has been moved out. This change, along with move to core V2 will be planned for the next release.